fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -318,8 +318,10 @@
      * The connection timeout and socket timeout config for thrift server
      * The value for thrift_client_timeout_ms is set to be larger than zero to prevent
      * some hang up problems in java.net.SocketInputStream.socketRead0
+     * 
```

Comment:
```
The connection timeout and socket timeout config for thrift server
The default value for thrift_client_timeout_ms is set to be zero to prevent readtimeout
```

 @ConfField public static int thrift_client_timeout_ms = 0;